api/reports/views.py

```diff
@@ -0,0 +1,69 @@
+# reports/views.py
+from django.contrib.auth import authenticate
+from django.http import HttpResponse
+from django.template import loader
+from django.shortcuts import render
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework import generics, permissions
+from rest_framework.authtoken.models import Token
+from rest_framework.decorators import api_view
+from rest_framework import status
+from rest_framework.response import Response
+
+from reports.models import Report
+from reports.permissions import IsOwner
+from reports.serializers import ReportSerializer
+
+
+def index(request):
+    latest_report_list = Report.objects.order_by('title')[:5]
+    context = {
+        'latest_report_list': latest_report_list,
+    }
+    return render(request, "index.html", context)
```

Comment:
Context in not being used anywhere. And can use `TemplateView` in urls rather than creating this view.